Example/Backpack/SwiftUI/Components/Badge/BadgeExampleVIew.swift

```diff
@@ -0,0 +1,78 @@
+//
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2022 Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+import Backpack_SwiftUI
+
+struct BadgeExampleVIew: View {
+    var body: some View {
+        ZStack {
+            Color(.skyGrayTint06.darkVariant(.black))
+                .ignoresSafeArea()
```

Comment:
no need to change it, but is this different than just saying
```swift
var body: some View {
  Hstack {
    regularBadges
    iconBadges
  }
   .background(BPKColor.backgroundColor) // or .skygrayTint6.darkVariant(.black)
}
```

Reply:
Doing it on the HSTack will only change the color of the stack, using the Color and a ZStack will produce a 'fullscreen' background color. 

I'm going to adjust the example to follow the Compose version, for alignment.